package.json

```diff
@@ -113,12 +112,13 @@
     "mydatepicker": "1.7.3",
     "ng2-bootstrap": "1.3.3",
     "ngx-dropdown": "0.0.22",
-    "ngx-modal": "0.0.29",
-    "ngx-fabric8-wit": "6.7.3",
+    "ngx-fabric8-wit": "0.0.0-development",
     "ngx-login-client": "0.4.3",
-    "ngx-widgets": "0.9.4",
+    "ngx-modal": "0.0.29",
+    "ngx-widgets": "0.0.0-development",
```

Comment:
Should ngx-widgets be 0.9.4? I tired 0.0.0-development and that doesn't install.